ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.component.ts

```diff
@@ -172,7 +172,12 @@ export class AddressInputComponent implements OnInit, OnDestroy {
     this.ais.inputIsReadOnly$.next(val);
   }
   @Input()
-  country: string | null;
+  set country(countryCode: string | null) {
+    this.ais.defaultCountryCode$.next(countryCode);
+  }
+  get country(): string | null {
+    return this.ais.defaultCountryCode$.getValue();
+  }
```

Comment:
Going to try to keep the state in the service instead of keeping a field variable here.